### PR TITLE
Removed errors from development console

### LINKED ISSUE #88: Remove all errors from development console
**Describe the bug**
Application has  a lots of console errors on the browser console window. We can fix it by doing some minor tweaks in code style.

## To Reproduce

1. Start the application with npm run start
2. Head over to console window once application is live on localhost
3. These are the errors and warnings we can get rid of.

## Expected behavior

There should 0 errors in the console.

## Environment (please complete the following information):

- OS: Ubuntu
- Browser: google-chrome-stable

### PR DESCRIPTION
**Pull request by rizwan2000rm**
**Issue Number**

fixes #88 

**Describe the changes you've made**

Removed all the development console errors. Turns out most of them were from a single line as a whole component was passed inside a <Typograpghy> tag which resulted in many errors.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`Failed prop type: Material-UI: This elevation 4 is not implemented
`

The error above is caused by soft Shadows in Material UI theming options. I have commented it out for now, as I don't understand what those options means maybe someone can help me with that.

**Additional context **

There are still 2 errors in the console which are due to some dependencies installed.

`[Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021.`

Not yet released will be fixed in a react update soon.
Issue: https://github.com/facebook/create-react-app/issues/10474
PR: https://github.com/facebook/react/pull/20831

`componentWillReceiveProps has been renamed, and is not recommended for use
`
Resolved and published in Alpha
PR: https://github.com/oliviertassinari/react-swipeable-views/pull/616

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 Please review this PR and also look into soft shadows theming options. I was not able to figure it out. But commenting it out removed the error from the console.

**Comment by ADRE9** [comment]
Hey @rizwan2000rm I mean I was also debugging the errors how did you find out the specific location to the errors from the chunk files.

File: src/theme/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import _ from "lodash";
 import { colors, createMuiTheme, responsiveFontSizes } from "@material-ui/core";
 import typography from "./typography";
-import { softShadows } from "./shadows";
+//! Adds a error Material-UI: This elevation `4` is not implemented.
+// import { softShadows } from "./shadows";
```
Comment (Abhishek-kumar09) [inline src/theme/index.js:5]: Please revert this, the shadows are not applied on the components, see the cards etc, the custom shadows are not applied. Have to look for some other way to fix it.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: src/theme/index.js (hunk 1 above)
Comment (rizwan2000rm) [reply src/theme/index.js]: @Abhishek-kumar09 Reverted.

**Comment by Abhishek-kumar09** [comment]
Github, rebase

**Comment by Abhishek-kumar09** [comment]
Github, rebase

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 I think that didn't work. Do you want me to rebase to a single commit ?

**Comment by Abhishek-kumar09** [comment]
I am seeing that you havn'e added anything for removing console errors, I could just see the eslint fixes done with `npm run lint:fix`. I could see many errors in the console.
Screenshot from 2021-03-18 13-09-09

**Comment by rizwan2000rm** [comment]
I'm sorry, I have confirmed with my fork again with latest changes, I don't see any of these errors. I have resolved them. All of those saying "X cannot appear as descendant of p" were due to the following line
` {value === index && (
        <Box>
          <Typography>{children}</Typography>
        </Box>
      )}`

at line 111, src/pages/CoursePageViewWithVideo/index.js 

`
{value === index && <Box>{children}</Box>}
`
which were resolved just by changing it to this. Typography uses p as a default tag which was wrapped around all those tags, hence resulting in so many errors. All other errors are due to some dependencies installed look above messge for details.

Logs

**Comment by Abhishek-kumar09** [comment]
Looks good, Ok to merge once CI passed

**Pull request merged**